src/unittest/python/pmf_tests.py

```diff
@@ -55,24 +55,40 @@ def test_fit_assigns_by_default(self):
     def test_fit_does_not_assign_if_requested(self):
         ps = pmf.PsychometricFunction(pmf.logistic, 0.5)
         self.assertIsNone(ps.params)
-        ps.fit(self.data, assign=False)
+        ps.ml_fit(self.data, assign=False)
         self.assertIsNone(ps.params)
         self.assertIsNone(ps.data)
 
     @mock.patch('esdt.pmf.optimize.fmin')
```

Comment:
I don't quite understand how decorators work yet, I assume it's supposed to be here though.

Reply:
Decorators are syntactic sugar for the following
```
@decorator 
def f(x):
    return x
```
instead of
```
def f(x):
    return x
f = decorator(f)
```
Both of those are the same. So you can use decorators to set things up before running a function. In this case, the set up is to replace the function `esdt.pmf.optimize.fmin` by a `mock.Mock()` object.